### DISCUSSION TITLE
Bezels (Overlay) app on off

### DISCUSSION THREAD

**Discussion by mlopezmad** [OnionUI/Onion#1352] [Ideas]
Bezel(Overlay)activation section in the app, activate or deactivate all system bezels, activate or deactivate bezel by system, or activate or deactivate the best aspect ratio on the screen.

**Comment by julenvitoria** [comment]
This would be very interesting!!! I've been practicing a little and I've managed to create a tweaks-style app, but I have the problem that I don't know how to make the option stay the same once you exit and re-enter... basically because it's the first time I've programmed in C. If someone could guide me maybe I could keep trying 😅
